modules/pel/peltool/README.md

```diff
@@ -9,6 +9,7 @@ To see all of peltool's capabilities, use the help option `peltool.py --help`.
 ## List PELs
 
 - Get list of PELs: `peltool.py -l` (By default serviceable PELs are listed)
+- Get list of all PELs: `peltool.py -lz`
```

Comment:
Get list of all PELs irrespective of PEL type

should we put it like this?

 same for line 46